### DISCUSSION TITLE
Support for "non-project files"?

### DISCUSSION THREAD

**Discussion by oblitum** [mfussenegger/nvim-jdtls#158] [Q&A] [answered]
Code at `lsp_clients.start` doesn't seem modeled at supporting standalone java file editing (https://github.com/neovim/neovim/pull/15430)?

eclipse.jdt.ls does provide basic support for them (so called "non-project files").

**Comment by mfussenegger** [comment, marked as answer]
Yes - jdtls currently is still compatible with neovim 0.5 and as far as I could tell there is little difference between the single file support and opening a file within a root_dir / workspaceFolder without a gradle/maven project so I didn't bother changing anything.
